libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -286,27 +285,37 @@ static IotMqttError_t _getIncomingPacket( void * pNetworkConnection,
                          pMqttConnection,
                          ( unsigned long ) pIncomingPacket->remainingLength,
                          ( unsigned long ) pIncomingPacket->type );
+                         
+            pIncomingPacket->pRemainingData = IotMqtt_MallocMessage( pIncomingPacket->remainingLength );
 
-            _flushPacket( pNetworkConnection, pMqttConnection, pIncomingPacket->remainingLength );
+            if( pIncomingPacket->pRemainingData == NULL )
+            {
+                IotLogError( "(MQTT connection %p) Failed to allocate buffer of length "
+                             "%lu for incoming packet type %lu.",
+                             pMqttConnection,
+                             ( unsigned long ) pIncomingPacket->remainingLength,
+                             ( unsigned long ) pIncomingPacket->type );
```

Comment:
please see the comment above

Reply:
Yes, that's the auto merge issue from earlier PR.

Reply:
Yea. Thought so